Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -291,6 +291,11 @@ theorem eq_symm_apply {α β} (e : α ≃ β) {x y} : y = e.symm x ↔ e y = x :
 theorem symm_bijective : Function.Bijective (Equiv.symm : (α ≃ β) → β ≃ α) :=
   Function.bijective_iff_has_inverse.mpr ⟨_, symm_symm, symm_symm⟩
 
+@[simp]
```

Comment:
It's not clear to me that this should be a `simp` lemma; can you find another example of a simp lemma assuming `Involutive`? We had trouble around this kind of thing long ago, but perhaps the reason is no longer relevant.

Reply:
I've found similar `simp` lemmas at `Equiv/Basic.lean`, such as `toPerm_symm`. This lemma is still nice to have since `toPerm_symm` isn't exactly what I want, so I will move it to that file.

Reply:
The simplifier will try to apply `toPerm_symm` only if it has `(h.toPerm f).symm`, while your lemma will match on every `Equiv.symm (e : α ≃ α)`.

Reply:
Ah, I thought it will check for the involutive condition too, I will remove the simp.

Reply:
It will match on `Equiv.symm _`, then try to prove `Involutive`.